demo/common.html

```diff
@@ -18,3 +18,9 @@
   ];
 </script>
 <link rel="import" href="../../elements-demo-resources/demo-navigation.html">
+
+<script>
```

Comment:
Currently `unresolved` is cleared even without this because `polymer.html` gets imported as a transitive dependency. However, as the dependencies will eventually drop import for `polymer.html`, this is required so the demos don't break.